### PR TITLE
{cae}[foss/2019b] waves2Foam v20200703

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3411.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/717ba2643f46be57f13f1bee56d2cf74 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3114.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/b38810e7acd2f154bfa476d6db1c32a5 for a full test report.

File: easybuild/easyconfigs/w/waves2Foam/waves2Foam-20200703-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('OpenFOAM', 'v1912'),
+    ('GSL', '2.6'),
+]
+
+local_oceanwave3d_optflags = "-ffpe-trap=invalid,zero,overflow -ffree-line-length-none -fstack-protector-all"
+local_oceanwave3d_common_opts = 'USER=easybuild LINLIB="-lopenblas -L%(installdir)s/lib -lskit" FC="$FC" '
```
Comment (lexming) [inline easybuild/easyconfigs/w/waves2Foam/waves2Foam-20200703-foss-2019b.eb:19]: What is the purpose of the `USER` variable? We do not have any user called `easybuild` in our system.

File: easybuild/easyconfigs/w/waves2Foam/waves2Foam-20200703-foss-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            (['OceanWave3D*/lib/libOceanWave3D.%s' % SHLIB_EXT], 'lib'),
+        ],
+    }),
+    (name, version, {
+        'easyblock': 'Binary',
+        # manual download required
+        # * visit https://sourceforge.net/p/openfoam-extend/svn/2140/tree/trunk/Breeder_1.6/other/waves2Foam/
+        # * click "Download Snapshot"
+        'sources': ['openfoam-extend-svn-r2140-trunk-Breeder_1.6-other-waves2Foam.zip'],
+        'checksums': ['ca2e018814c0635b69868e3586f61cebf7eebb1514488a45a525f90e49ab41f4'],
```
Comment (zao) [inline easybuild/easyconfigs/w/waves2Foam/waves2Foam-20200703-foss-2019b.eb:80]: This checksum does not seem stable, following the instructions I got:
`f5dad792ecfce73ac6a254ccf8b2b43a75a6a6e84bd221836c349d4cbe413761 *openfoam-extend-svn-r2140-trunk-Breeder_1.6-other-waves2Foam.zip`

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
freja - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 3.8.2
See https://gist.github.com/cf52cfedf849187eee771e306ebbc657 for a full test report.

**Comment by zao** [comment]
> Test report by @zao
> **SUCCESS**

This build report had to `--ignore-checksums` to deal with the unstable zip.

**Comment by verdurin** [comment]
@boegel could you take a look at the issues raised here, please?

**Comment by akesandgren** [comment]
@boegel ping on this one

**Comment by akesandgren** [comment]
Closing this PR, since it (only) uses deprecated toolchain versions,
see https://docs.easybuild.io/policies/toolchains .

**Pull request closed without merging**